profile-widget/profilewidget2.cpp

```diff
@@ -528,6 +528,7 @@ void ProfileWidget2::resetZoom()
 // Currently just one dive, but the plan is to enable All of the selected dives.
 void ProfileWidget2::plotDive(struct dive *d, bool force, bool doClearPictures)
 {
+	Q_UNUSED(doClearPictures);
```

Comment:
Not correct for desktop -> this parameter is used.

Reply:
I know that, but I thought it was pure overkill to add a #ifdef IOS for just reducing noise. I can of course do that if so wished

Reply:
i'm fine with the `Q_UNUSED` being added even if it's only the case on `SUBSURFACE_MOBILE`.

Reply:
you can also do the following near the prologue:
```
{
#ifndef SUBSURFACE_MOBILE
	QTime measureDuration; // let's measure how long this takes us (maybe we'll turn of TTL calculation later
	measureDuration.start();
#else
	Q_UNUSED(doClearPictures);
#endif
	static bool firstCall = true;
```

Reply:
Of course I must have been sleeping (it was before my morning coffee) all corrected.